packages/@contentful--app-scripts/lib/cache-credential/index.js

```diff
@@ -0,0 +1,89 @@
+const dotenv = require('dotenv');
+const fs = require('fs');
+const { EOL } = require('os');
+const chalk = require('chalk');
+
+
+const fsPromises = fs.promises;
```

Comment:
I don't think there's a need to drag `Promises` around if we don't use the old API, just name it `fs`

Reply:
We do use the old api for `fs.constants`. We can extract both and rename but wanted to avoid confusion considering most projects just call it `fsPromises`